diff --git a/lib/esm/handlers/search/Configuration.js b/lib/esm/handlers/search/Configuration.js
index e57a506567da593a2a37d1f638f81878f96d5922..42ce1fec75f578a44665e3a813015bd4d2e72a5f 100644
--- a/lib/esm/handlers/search/Configuration.js
+++ b/lib/esm/handlers/search/Configuration.js
@@ -153,7 +153,7 @@ function buildConfiguration({ state, queryConfig, cloud, host, index, apiKey, he
         index: index,
         connectionOptions: {
             apiKey: apiKey,
-            headers: Object.assign(Object.assign({}, additionalHeaders), { "x-elastic-client-meta": metaHeader })
+            headers: Object.assign(Object.assign({}, additionalHeaders), {})
         },
         hits: {
             fields: hitFields,
