diff --git a/node_modules/@outlinestudio/designtokenscss/dist/cjs/utils/actions.js b/node_modules/@outlinestudio/designtokenscss/dist/cjs/utils/actions.js
index 27bc513..534d400 100644
--- a/node_modules/@outlinestudio/designtokenscss/dist/cjs/utils/actions.js
+++ b/node_modules/@outlinestudio/designtokenscss/dist/cjs/utils/actions.js
@@ -40,5 +40,5 @@ function MountedVariables(theme) {
     }
 
     return "--".concat(item, ": ").concat(theme.tokens[item]);
-  }).toString().replace(/,/g, ';');
+  }).join('; ');
 }
diff --git a/node_modules/@outlinestudio/designtokenscss/dist/esm/utils/actions.js b/node_modules/@outlinestudio/designtokenscss/dist/esm/utils/actions.js
index 6ee5c58..9f7d357 100644
--- a/node_modules/@outlinestudio/designtokenscss/dist/esm/utils/actions.js
+++ b/node_modules/@outlinestudio/designtokenscss/dist/esm/utils/actions.js
@@ -30,5 +30,5 @@ export function MountedVariables(theme) {
     }
 
     return "--".concat(item, ": ").concat(theme.tokens[item]);
-  }).toString().replace(/,/g, ';');
+  }).join('; ')
 }
