diff --git a/node_modules/axios/lib/utils.js b/node_modules/axios/lib/utils.js
index 83eb1c6..8aa50e8 100644
--- a/node_modules/axios/lib/utils.js
+++ b/node_modules/axios/lib/utils.js
@@ -112,12 +112,25 @@ function isObject(val) {
  * @return {boolean} True if value is a plain Object, otherwise false
  */
 function isPlainObject(val) {
-  if (toString.call(val) !== '[object Object]') {
+  // copied from https://github.com/jonschlinkert/is-plain-object
+  var ctor,prot;
+
+  if (isObject(val) === false) return false;
+
+  // If has modified constructor
+  ctor = val.constructor;
+  if (ctor === undefined) return true;
+
+  // If has modified prototype
+  prot = ctor.prototype;
+  if (isObject(prot) === false) return false;
+
+  // If constructor does not have an Object-specific method
+  if (prot.hasOwnProperty('isPrototypeOf') === false) {
     return false;
   }
-
-  var prototype = Object.getPrototypeOf(val);
-  return prototype === null || prototype === Object.prototype;
+  // Most likely a plain Object
+  return true;
 }
 
 /**
