diff --git a/dist/cjs/index.d.ts b/dist/cjs/index.d.ts
index 9ed199e97917a48ebe87a33625503bc5295de2e0..b7d9ff6d4b3d0dacc448d9518ed4d3dc1580f1ea 100644
--- a/dist/cjs/index.d.ts
+++ b/dist/cjs/index.d.ts
@@ -4,4 +4,3 @@ export declare type Htmlparser2TreeAdapterMap = TreeAdapterTypeMap<AnyNode, Pare
 /** @internal */
 export declare function serializeDoctypeContent(name: string, publicId: string, systemId: string): string;
 export declare const adapter: TreeAdapter<Htmlparser2TreeAdapterMap>;
-//# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --git a/dist/cjs/index.js b/dist/cjs/index.js
index 63634618969ee6659d739c4aaa1c404358175299..002cbe88a3bdd4afe589b7978d58cf61f5ff27ae 100644
--- a/dist/cjs/index.js
+++ b/dist/cjs/index.js
@@ -212,4 +212,3 @@ exports.adapter = {
         node.sourceCodeLocation = Object.assign(Object.assign({}, node.sourceCodeLocation), endLocation);
     },
 };
-//# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/dist/index.d.ts b/dist/index.d.ts
index 9ed199e97917a48ebe87a33625503bc5295de2e0..b7d9ff6d4b3d0dacc448d9518ed4d3dc1580f1ea 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -4,4 +4,3 @@ export declare type Htmlparser2TreeAdapterMap = TreeAdapterTypeMap<AnyNode, Pare
 /** @internal */
 export declare function serializeDoctypeContent(name: string, publicId: string, systemId: string): string;
 export declare const adapter: TreeAdapter<Htmlparser2TreeAdapterMap>;
-//# sourceMappingURL=index.d.ts.map
\ No newline at end of file
diff --git a/dist/index.js b/dist/index.js
index 655f970ea92b76caa08d164e9f97f6165dcd15d6..10d7db38b7c384f5eb57e4d685051aef408a129f 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -211,4 +211,3 @@ export const adapter = {
         };
     },
 };
-//# sourceMappingURL=index.js.map
