diff --git a/node_modules/vite-plugin-vue-svg/index.js b/node_modules/vite-plugin-vue-svg/index.js
index 8a4f47d..ce49cfa 100644
--- a/node_modules/vite-plugin-vue-svg/index.js
+++ b/node_modules/vite-plugin-vue-svg/index.js
@@ -53,7 +53,7 @@ module.exports = (options = {}) => {
             result = await compileSvg(svg, idWithoutQuery);
 
             if (isBuild) {
-              cache.set(path, result);
+              cache.set(idWithoutQuery, result);
             }
           }
 
