diff --git a/node_modules/vite-svg-loader/index.js b/node_modules/vite-svg-loader/index.js
index 133df06..3caa73d 100644
--- a/node_modules/vite-svg-loader/index.js
+++ b/node_modules/vite-svg-loader/index.js
@@ -32,14 +32,14 @@ module.exports = function svgLoader (options = {}) {
         return // File couldn't be loaded, fallback to default loader
       }

-      if (importType === 'raw') {
-        return `export default ${JSON.stringify(svg)}`
-      }
-
       if (svgo !== false) {
         svg = optimizeSvg(svg, svgoConfig).data
       }

+      if (importType === 'raw') {
+        return `export default ${JSON.stringify(svg)}`
+      }
+
       const { code } = compileTemplate({
         id: JSON.stringify(id),
         source: svg,
