diff --git a/index.js b/index.js
index 87cc6c97e86bddc8d050f18690cdbfaea6375846..00bc2c2dbf4ebf13180c45fba86050810ad0ee1c 100644
--- a/index.js
+++ b/index.js
@@ -33,10 +33,6 @@ module.exports = function svgLoader (options = {}) {
         return
       }
 
-      if (importType === 'raw') {
-        return `export default ${JSON.stringify(svg)}`
-      }
-
       if (svgo !== false && query !== 'skipsvgo') {
         svg = optimizeSvg(svg, {
           ...svgoConfig,
@@ -44,6 +40,10 @@ module.exports = function svgLoader (options = {}) {
         }).data
       }
 
+      if (importType === 'raw') {
+        return `export default ${JSON.stringify(svg)}`
+      }
+
       const { code } = compileTemplate({
         id: JSON.stringify(id),
         source: svg,