diff --git a/node_modules/vite-svg-loader/index.js b/node_modules/vite-svg-loader/index.js
index c2f93f3..00bc2c2 100644
--- a/node_modules/vite-svg-loader/index.js
+++ b/node_modules/vite-svg-loader/index.js
@@ -33,10 +33,6 @@ module.exports = function svgLoader (options = {}) {
         return
       }
 
-      if (importType === 'raw') {
-        return `export default ${JSON.stringify(svg)}`
-      }
-
       if (svgo !== false && query !== 'skipsvgo') {
         svg = optimizeSvg(svg, {
           ...svgoConfig,
@@ -44,6 +40,10 @@ module.exports = function svgLoader (options = {}) {
         }).data
       }
 
+      if (importType === 'raw') {
+        return `export default ${JSON.stringify(svg)}`
+      }
+
       const { code } = compileTemplate({
         id: JSON.stringify(id),
         source: svg,
